crates/cheatcodes/spec/src/vm.rs

```diff
@@ -36,6 +36,24 @@ interface Vm {
         RecurrentPrank,
     }
 
+    /// The kind of account access that occurred.
+    enum AccountAccessKind {
```

Comment:
Nit, but I think we should lowercase everything here and in the solidity definitions? e.g. `Delegatecall`, Callcode`, `Staticcall`, `Selfdestruct`? I feel like I don't usually see those words cased in the current way, but happy to go with whatever is consistent with the rest of the codebase/revm

Reply:
leaving it capital cased for consistency. The `CallerMode` enums use capital case.

Reply:
Ah you're right this matches revm: https://github.com/bluealloy/revm/blob/aa5667764890bc554ea6392ea1e84e3e1991be6d/crates/interpreter/src/inner_models.rs#L63-L72

And Safe: https://github.com/safe-global/safe-contracts/blob/915045fcf7e62cc4564c1755b2676b8a2ab22b10/contracts/common/Enum.sol#L11

So, marking this resolved